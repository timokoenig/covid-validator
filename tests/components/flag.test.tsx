import React from 'react'
import renderer from 'react-test-renderer'
import Flag from '../../components/flag'

it('renders correctly', () => {
  const tree = renderer.create(<Flag country="de" size={25} />).toJSON()
  expect(tree).toMatchSnapshot()
})
