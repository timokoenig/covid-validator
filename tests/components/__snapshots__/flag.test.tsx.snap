// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="fi fi-de fis"
  style={
    Object {
      "borderRadius": 12.5,
      "height": 25,
      "width": 25,
    }
  }
/>
`;
